apps/src/p5lab/gamelab/GameLab.js

```diff
@@ -69,6 +73,26 @@ export default class GameLab extends P5Lab {
   }
 
   runButtonClick() {
+    // For AI Rubrics Pilot
+    if (
+      experiments.isEnabled('ai-rubrics') &&
+      hasScriptData('script[data-rubricdata]')
+    ) {
+      const rubricData = getScriptData('rubricdata');
+      const script = document.querySelector('script[data-level]');
+      const config = JSON.parse(script.dataset.level);
```

Comment:
Instead of getting this out of the DOM again, does `this.level` (set [here](https://github.com/code-dot-org/code-dot-org/blob/8f06a1bcd1e2896804e4682e2b698da6c036a084/apps/src/p5lab/P5Lab.js#L236C5-L236C31)) give you what you want?